amulet/sentry.py

```diff
@@ -254,15 +255,16 @@ def wait_for_status(self, juju_env, services, timeout=300):
             raise
 
     def wait(self, timeout=300):
-        #for unit in self.unit:
         ready = False
         try:
+            now = time.time()
             with helpers.timeout(timeout):
                 # Make sure we're in a 'started' state across the board
```

Comment:
The previous code does not actually check for "started" here.  I would recommend removing this comment to reflect the code.